packages/core/src/startup/StartupSystem.ts

```diff
@@ -1,21 +1,13 @@
 import { extensions, ExtensionType } from '@pixi/extensions';
 
 import type { ExtensionMetadata } from '@pixi/extensions';
-import type { BackgroundOptions } from '../background/BackgroundSystem';
-import type { IRenderer } from '../IRenderer';
+import type { IRenderer, IRendererOptions } from '../IRenderer';
 import type { IRendererPlugins } from '../plugin/PluginSystem';
 import type { ISystem } from '../system/ISystem';
-import type { ContextOptions } from '../systems';
-import type { ViewOptions } from '../view/ViewSystem';
 
-// TODO this can be infered by good use of generics in the future..
-export interface StartupOptions extends Record<string, unknown>
+export interface StartupOptions extends IRendererOptions
```

Comment:
Overall, this change is great. Simplification and decentralization is awesome.

The only thing that I might improve, (and totally fine to do this as a follow up PR, if you think it's out of scope):

Organizationally, it would be nice if type concerns were isolated for IRendererOptions. For instance, BackgroundSystem should "own" the option types for background, backgroundAlpha, bsckgroundColor. to make it easier to connect those options to where they are used. Perhaps IRendererOptions could be a union of things like BackgroundSystemOptions, ContextSystemOptions, etc instead of having them centralized in IRendererOptions. In the future we'd like to make systems more composable, this would help that.

Reply:
Maybe with `GlobalMixins.IRendererOptions`?

Reply:
I think GlobalMixins is more about shared/external package typings. Most of this stuff is internal-core stuff so probably not necessary.